android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -56,6 +58,19 @@ public void onMapReady(GoogleMap map) {
                     CameraPosition.builder().target(location).zoom(15f).bearing(0).tilt(0).build()));
 
             mMap = map;
+
+            mMap.getUiSettings().setMapToolbarEnabled(true);
+            mMap.setOnMarkerClickListener(new GoogleMap.OnMarkerClickListener() {
```

Comment:
No need to set on marker click listener & implement intent on map because on clicking on marker map will show detail of marker and maptoolbar will be visible.

It will not be good on clicking marker it directly opens map it should first display marker details

![mapmarker](https://cloud.githubusercontent.com/assets/18264684/25836820/93f17522-34a7-11e7-9cd7-5d9d972d4899.jpg)